src/panels/scene-viewer/SceneViewer.tsx

```diff
@@ -267,20 +268,38 @@ export const SceneViewer = (props: SceneViewerPropsFromParent) => {
   const staticPluginPath = useMemo(() => `public/plugins/${plugin.id}`, []);
 
   const viewerConfig = useMemo(() => {
+    const interval = props.options.queryRefreshInterval !== undefined ? props.options.queryRefreshInterval * 1000 : undefined;
+
     return {
       dracoDecoder: {
         enable: true,
         path: `${window.location.origin}/${staticPluginPath}/static/draco/`,
       },
+      featureConfig: {
+        AutoQuery: true,
+      },
+      dataBindingQueryRefreshRate: interval
     };
-  }, [staticPluginPath]);
+  }, [staticPluginPath, props.options.queryRefreshInterval]);
 
   const viewport: Viewport = useMemo(() => {
+    const from = props.data.timeRange.from.valueOf();
+    const to = props.data.timeRange.to.valueOf();
+    const fromRaw = props.timeRange.raw.from;
+    const toRaw = props.timeRange.raw.to;
+
+    if (typeof fromRaw === 'string' && typeof toRaw === 'string' && fromRaw.includes('now') && toRaw.includes('now')) {
```

Comment:
What's the difference between the regular range and raw range?

Reply:
for relative time rage, we see 'now-5m', but the regular range will always show the calculated absolute date value